images/openvswitch/Dockerfile

```diff
@@ -3,16 +3,12 @@
 #
 # The standard name for this image is openshift/openvswitch
 #
-FROM openshift/origin-base
+FROM openshift/node
 
 COPY scripts/* /usr/local/bin/
 
-RUN curl -L -o /etc/yum.repos.d/origin-next-epel-7.repo https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/maxamillion-origin-next-epel-7.repo && \
-    INSTALL_PKGS="openvswitch" && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
-    yum clean all && \
-    chmod +x /usr/local/bin/*
+RUN INSTALL_PKGS="openvswitch" && \
```

Comment:
You might have already noticed this, but I'm just going to put it in as a comment so others will.
Yes, openvswitch is already in node, or it should be.  This little blip isn't installing openvswitch, it is verifying that it's there.
So this is a nice check incase we manage to grab a node image that doesn't have openvswitch for some reason.